next-hardhat/README.md

```diff
@@ -8,83 +8,39 @@ It also features multiple files and different entry points by resolving multiple
 
-```bash
-npm i
-```
-
-⚠️  The examples install specific versions of the required `@noir-lang` npm pacakges. Different versions are not cross compatible. Do not use `yarn` as it fails to grab the NPM specific versions we need. Noir is being actively developed, so the latest versions of various packages are often incompatible with each other. ⚠️
-
-2. Write circuits in `./circuits/src`.
-
-You can read more about writing circuits in Noir on the [Noir docs](https://noir-lang.org/).
-
-3. Create the verifier contract
-
-Although the `npm build` already generates and compiles the circuits before deploying, you can manually create the Solidity verifier contract:
-
-- natively by navigating to the `.circuits/` directory and running `nargo codegen-verifier`. If you
-  generate the verifier contract via this method, you may need to copy the file created at
-  `./circuits/contract/plonk_vk.sol` to the hardhat contract directory at `./contract`.
-- with wasm by running the `genContract.ts` script:
+2. Install dependencies with
 
 ```bash
-npx ts-node scripts/genContract.ts
+yarn
 ```
 
-4. Create proofs
-
-**Natively**
-
-In `./circuits`:
-
-- Compile your circuits if you haven't already with `nargo compile <CIRCUIT_NAME>`.
-- Populate the inputs in `Prover.toml`
-- Generate proof with `nargo prove <proof_name>`
+3. Write circuits in `./circuits/src`.
 
-**With Typescript and WASM**
-
-- See an example React component in [`./components/component.tsx`](./components/component.tsx)
-
-5. Verify proofs
-
-**Natively**
+You can read more about writing circuits in Noir on the [Noir docs](https://noir-lang.org/).
 
-In `./circuits`:
+4. Create the verifier contract
 
-- Verify proof with `nargo verify <proof_name>`
+Navigate to the `.circuits/` directory and run `nargo codegen-verifier`
 
-**With Typescript and WASM**
+5. Compile your circuit
 
-- See the example in [`./components/component.tsx`](./components/component.tsx)
+Compile your circuits with `nargo compile main`.
 
 6. Deploy
 
 - Start a local development EVM at <http://localhost:8545>, for example with `npx hardhat node`.
 - Copy `./.env.example` to `./.env` and add keys for alchemy (to act as a node) and the deployer's private key. Make sure you have funds in this account.
-- Run `NETWORK=localhost npm build` to build the project and deploy contracts to the local development chain
+- Run `NETWORK=localhost yarn build` to build the project and deploy contracts to the local development chain
 
-You can choose any other network in `hardhat.config.ts` and deploy there using this `NETWORK` environment variable. For example, `NETWORK=mumbai npm build` or `NETWORK=sepolia npm build`. Feel free to contribute with other networks in `hardhat.config.ts`
+You can choose any other network in `hardhat.config.ts` and deploy there using this `NETWORK` environment variable. For example, `NETWORK=mumbai yarn build` or `NETWORK=sepolia yarn build`. Feel free to contribute with other networks in `hardhat.config.ts`
```

Comment:
Just need to add a line, "Once your contracts are deployed and the build is finished, start the web app with `yarn start`."